### PR TITLE
Add new Query for GeoLocation

### PR DESCRIPTION
**Pull request by vnegrisolo**
- [x] use gauss function to query by location
- [x] parse it GeoLocation function in Query class

### CODE REVIEW

File: lib/elasticband/query/score_function/gauss.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Elasticband
+  class Query
+    class ScoreFunction
+      class Gauss < ScoreFunction
```
Comment (brennovich) [inline lib/elasticband/query/score_function/gauss.rb:4]: Gauss

**Comment by brennovich** [comment]
:+1:

**Comment by rubensmabueno** [comment]
:+1:

**Comment by rubensmabueno** [comment]
:-1:

**Comment by rubensmabueno** [comment]
You forgot to increment the version.

**Comment by vnegrisolo** [comment]
done @brennovich @rubensmabueno

**Comment by rubensmabueno** [comment]
:+1:

**Comment by brennovich** [comment]
:+1:

**Pull request merged**